### ISSUE TITLE
Text in AGS windows fuzzy in places

### ISSUE THREAD

**Issue by Chaitran77** [koeqaife/hyprland-material-you#45]
All text in the settings window becomes fuzzy when it is active (mouse hovering over it) and goes back in "focus" when it is inactive. The top bar has fuzzy fonts in some places and not in others (time/date, window switcher and battery widgets are clear, rest are fuzzy) and the app menu has mostly fuzzy fonts.

Please could I have some help fixing this? Screenshots are below (I couldn't capture the settings app being fuzzy because it only happens on hover).

Top bar:
![image](https://github.com/user-attachments/assets/0ccbf9d3-285f-44ce-94bb-454c4067669a)

App menu:
![image](https://github.com/user-attachments/assets/c1101cfc-bd05-4679-8be7-f3275ea6fe3b)

**Comment by koeqaife** [comment]
have you install material icons?

**Comment by koeqaife** [comment]
install ttf-material-symbols-variable-git from AUR

**Comment by koeqaife** [comment]
and restart pc

**Comment by Chaitran77** [comment]
Installed and restarted but text is still fuzzy in the same places. Icons appeared but that resolves a separate issue.

**Comment by koeqaife** [comment]
start ~/setup/copy.sh

**Comment by Chaitran77** [comment]
Yep that worked! Changed into the home directory and executed as `./dotfiles/setup/copy.sh`. Didn't even have to restart.

Btw I hope you are integrating these solutions into your repo! Thanks for the help!

**Comment by koeqaife** [comment]
I think you clicked no at some stages of the installation, did you when you used the script it told you "please restart your computer"?

**Comment by Chaitran77** [comment]
It did tell me to restart and I have restarted multiple times since

**Comment by koeqaife** [comment]
Okay, then the problem may be due to the fact that one person changed the installation script recently

**Comment by koeqaife** [comment]
Okay, then the problem may be due to the fact that one person changed the installation script recently

**Comment by Chaitran77** [comment]
Hmm. I do have many other issues occuring such as the weather not showing despite the API key and location being entered, backlight and brightness sliders not doing anything despite keys working, color picker not launching, night light not doing anything, CPU temp showing undefined, etc...

Sorry I know it's a lot but if you have any intel on this I would appreciate the help.

**Comment by koeqaife** [comment]
It works fine for many people, I don't know why you are having such problems

**Comment by koeqaife** [comment]
try to run `~/dotfiles/install.sh packages` btw

**Comment by Chaitran77** [comment]
Thank you for that, it helped me figure out what had happened: I already had some packages installed (sddm-git, the modded version of rofi, python-pywal) and when the question `Package 1 and package 2 are in conflict. Remove package 2? [y/N]` is output, I think the script sends an enter key or something similar which is interpreted as a `n`, causing the installation of that package not to proceed and for the script to just continue when running the full installation (i.e. without the `packages` argument)

**Comment by Chaitran77** [comment]
I restarted and the fuzzy text and brightness slider are now fixed.

**Comment by Chaitran77** [comment]
I restarted and the fuzzy text and brightness slider are now fixed. Thanks for your help.
